provision/vvv-init.sh

```diff
@@ -240,6 +240,11 @@ else
   else
     update_wp
   fi
+  cat <<EOF > "${PUBLIC_DIR_PATH}/wp-cli.yml"
+@${VVV_SITE_NAME}:
+  ssh: vagrant@192.168.50.4
```

Comment:
not every VVV VM has this IP, I believe Hyper-V doesn't respect VVV's IP choice, it should be `vagrant@vvv.test`